### ISSUE TITLE
Error when using string in command

### ISSUE THREAD

**Issue by jlwoolf** [JanDeDobbeleer/oh-my-posh#1974]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

When writing out a command in the terminal if I use double quotes without closing the quotes I get this warning:

`
Error: invalid argument "" for "oh-my-posh prompt print" Usage:   oh-my-posh prompt print [debug|primary|secondary|transient|right|tooltip|valid|error] [flags] System.Management.Automation.RemoteException Flags:       --command string         tooltip command   -e, --error int              last exit code       --eval
   output the prompt for eval       --execution-time float   timing of the last command   -h, --help                   help for print   -p, --plain                  plain text output (no ANSI)       --pswd string            current working directory (according to pwsh)       --pwd string             current working directory       --shell string           the shell to print for   -s, --stack-count int        number of locations on the stack   -w, --terminal-width int     width of the terminal System.Management.Automation.RemoteException Global Flags:   -c, --config string   config (required) System.Management.Automation.RemoteException invalid argument "" for "oh-my-posh prompt print"
`

I've included a video of typing an example command with the weird behavior. It seems to happen with a space following a double quote.

https://user-images.githubusercontent.com/60640764/159774196-96013ec4-80ed-4fff-9e68-0b0999b71660.mp4

### Theme

Default, jandedobbeleer, powerlevel10k

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```shell
Version: 7.46.1

Segments:

ConsoleTitle(true) -  10 ms - pwsh in ~
session(true)  -   0 ms -  jlwoolf 
path(true)     -   0 ms -   ~
git(false)     -   1 ms -
root(false)    -   0 ms -
exit(true)     -   0 ms -   
node(false)    -   0 ms -
go(false)      -   0 ms -
python(false)  -   0 ms -
shell(true)    -   0 ms - in pwsh
time(true)     -   0 ms - at 12:48:45

Run duration: 13.7163ms

Cache path: C:\Users\jlwoolf\AppData\Local\oh-my-posh

Logs:

2022/03/23 12:48:45 Flags duration: 0s, args:
2022/03/23 12:48:45 Flags duration: 0s, args:
2022/03/23 12:48:45 debug: Getenv

2022/03/23 12:48:45 Getenv duration: 0s, args: OMP_CACHE_DISABLED
2022/03/23 12:48:45 Root duration: 0s, args:
2022/03/23 12:48:45 Shell duration: 10.1547ms, args:
2022/03/23 12:48:45 ErrorCode duration: 0s, args:
2022/03/23 12:48:45 IsWsl duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 User duration: 0s, args:
2022/03/23 12:48:45 Host duration: 0s, args:
2022/03/23 12:48:45 GOOS duration: 0s, args:
2022/03/23 12:48:45 TemplateCache duration: 10.1547ms, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Getenv

2022/03/23 12:48:45 Getenv duration: 0s, args: SSH_CONNECTION
2022/03/23 12:48:45 debug: Getenv

2022/03/23 12:48:45 Getenv duration: 0s, args: SSH_CLIENT
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 Flags duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 GOOS duration: 0s, args:
2022/03/23 12:48:45 GOOS duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 GOOS duration: 0s, args:
2022/03/23 12:48:45 IsWsl duration: 0s, args:
2022/03/23 12:48:45 StackCount duration: 0s, args:
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 GOOS duration: 0s, args:
2022/03/23 12:48:45 HasCommand duration: 1.1924ms, args: git.exe
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 error: HasParentFilePath
CreateFile C:\.git: The system cannot find the file specified.
2022/03/23 12:48:45 HasParentFilePath duration: 0s, args: .git
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 Root duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 ErrorCode duration: 0s, args:
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 0s, args: *.py
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 599.8µs, args: *.ipynb
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 0s, args: pyproject.toml
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 0s, args: venv.bak
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 0s, args: venv
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 PathSeparator duration: 0s, args:
2022/03/23 12:48:45 HasFiles duration: 0s, args: .venv
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 Shell duration: 0s, args:
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 debug: Pwd
C:\Users\jlwoolf
2022/03/23 12:48:45 Pwd duration: 0s, args:
2022/03/23 12:48:45 debug: Home
C:\Users\jlwoolf
2022/03/23 12:48:45 TemplateCache duration: 0s, args:
2022/03/23 12:48:45 debug: Getenv
C:\Users\jlwoolf\AppData\Local
2022/03/23 12:48:45 Getenv duration: 0s, args: LOCALAPPDATA
2022/03/23 12:48:45 CachePath duration: 1.0533ms, args:
```

**Comment by JanDeDobbeleer** [comment]
@jlwoolf is it correct in stating this only happens when tooltips are enabled? I noticed this on fish earlier today and fixed it there, this could be the same issue (but the on PowerShell side)

**Comment by jlwoolf** [comment]
Yes. Only with tooltips enabled.

**Comment by JanDeDobbeleer** [comment]
@jlwoolf this was fixed in [7.46.2](https://github.com/JanDeDobbeleer/oh-my-posh/releases/tag/v7.46.2) :-)
